core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form class="form-inline" ng-submit="addExploration()">
-  <input class="form-control protractor-test-add-exploration-input" ng-model="newExplorationId" type="text" placeholder="Existing exploration ID">
-  <button class="btn btn-default protractor-test-add-exploration-button" type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-control protractor-test-add-exploration-input" ng-model="newExplorationId" type="text" placeholder="Search explorations by name" typeahead="exploration for exploration in fetchExplorationsMetadata($viewValue)">
```

Comment:
Umm, I meant that -- It adds the exploration to the collection when we just type an ID and press enter. It is not searchable using ID.